doc/plugins/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _plugins:
+
+Plugins
+================================================================================
+
+What is a Scopy plugin ?
+
+General IIO plugins
+
+* ADC Plugin
+
+* DAC plugin
+
+* :ref:`Datalogger <datalogger>`
+
+* Debugger
+
+* :ref:`Register Map <registermap>`
+
+Application Specific plugins
+
+* ADALM2000 plugins :
+
+    * :ref:`Oscilloscope <oscilloscope>`
```

Comment:
For nesting with spaces, you always align the space with the first char of the last line
```suggestion
* ADALM2000 plugins :

  * :ref:`Oscilloscope <oscilloscope>`
```